File: C++/Code/Aksesoris.cpp
#include <iostream>
#include <string>
#include "PetShop.cpp"
using namespace std;

class Aksesoris : public PetShop
{
private:
    string jenis;
    string bahan;
    string warna;

public:
    Aksesoris() {

    }

    Aksesoris(string id, string nama_produk, int harga_produk, int stok_produk, string jenis, string bahan, string warna) : PetShop(id, nama_produk, harga_produk, stok_produk) {
        this->jenis = jenis;
        this->bahan = bahan;
        this->warna = warna;
    }

    string getJenis() {
        return this->jenis;
    }

    string getBahan() {
        return this->bahan;
    }

    string getWarna() {
        return this->warna;
    }

    ~Aksesoris(){

    }
};

File: PHP/Code/Aksesoris.php
<?php
require_once "PetShop.php";

class Aksesoris extends PetShop {
    private $jenis = "";
    private $bahan = "";
    private $warna = "";

    public function __construct($id, $nama_produk, $harga_produk, $stok_produk, $gambar, $jenis, $bahan, $warna) {
        parent::__construct($id, $nama_produk, $harga_produk, $stok_produk, $gambar);
        $this->jenis = $jenis;
        $this->bahan = $bahan;
        $this->warna = $warna;
    }

    public function getJenis() {
        return $this->jenis;
    }

    public function getBahan() {
        return $this->bahan;
    }

    public function getWarna() {
        return $this->warna;
    }
}

?>